common/src/main/templates/io/netty/util/collection/KObjectMap.template

```diff
@@ -76,37 +49,11 @@ public interface @K@ObjectMap<V> {
     V remove(@k@ key);
 
     /**
-     * Returns the number of entries contained in this map.
-     */
-    int size();
-
-    /**
-     * Indicates whether or not this map is empty (i.e {@link #size()} == {@code 0]).
-
-     */
-    boolean isEmpty();
-
-    /**
-     * Clears all entries from this map.
-     */
-    void clear();
-
-    /**
      * Indicates whether or not this map contains a value for the specified key.
      */
     boolean containsKey(@k@ key);
 
     /**
-     * Indicates whether or not the map contains the specified value.
-     */
-    boolean containsValue(V value);
-
-    /**
-     * Gets an iterable collection of the entries contained in this map.
-     */
-    Iterable<Entry<V>> entries();
-
-    /**
      * Gets the keys contained in this map.
      */
     @k@[] keys();
```

Comment:
Now that we no longer have `entries()` this option seems like the only way to "iterate" over the native collection.  This method may be a bit intensive and require a bunch of memory allocation.  Is it worth supporting this at the interface level?